dependencies.yaml

```diff
@@ -451,3 +457,16 @@ dependencies:
             packages:
               - cupy-cuda11x>=12.0.0
           - {matrix: null, packages: [cupy-cuda11x>=12.0.0]}
+  test_libcuspatial:
+    common:
+      - output_types: conda
+        packages:
+          - libcuspatial==24.6.*
+          - libcuspatial-tests==24.6.*
+  test_cuspatial:
+    common:
+      - output_types: conda
+        packages:
+          - libcuspatial==24.6.*
+          - cuspatial==24.6.*
+          - cuproj==24.6.*
```

Comment:
```suggestion
          - cuproj==24.6.*

```

Reply:
PS - best to wait for this current build to finish, especially the docs part. The tests look pretty good so far.

Reply:
I guess we don't have a yaml linter that catches these in pre-commit?

Reply:
Looks like we could enable [`end-of-file-fixer` with `pre-commit`]( https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#end-of-file-fixer )